local sensors API support

bpowers opened the pull request:
This adds support for the local sensors API that individual devices
expose.  I tried a few different approaches, but it seems like a
distinct top level entrypoint and subclassing of `AwairDevice` was the
cleanest approach here.  It has been... a while since I've done much
Python, happy to iterate on this.

Fixes #8

bpowers commented:
all lints now pass, and the one failing unit test (which was never passing for me) is separately fixed in #10

ahayworth commented:
@bpowers Yay! I'm excited that the local API is ready for use! 😄 

This is my bad - but I didn't have CI set up to run the linters on PRs, and it looks like `black` wants to reformat something. Can you run `black .` in the PR and push whatever it wants to reformat?

Once the linters are all happy, we can merge this and cut a new release! :D

bpowers commented:
@ahayworth super!  I rebased, ran black, and squashed a few of the commits together for a nicer history, but didn't make any other changes.  Let me know if anything else slipped through!

ahayworth commented:
🎉 🎉 🎉 
```
~/projects/python_awair bpowers/local-sensors-api
env ❯ python
Python 3.8.5 (default, Jul 28 2020, 09:26:19)
[Clang 11.0.3 (clang-1103.0.32.62)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import asyncio
>>> import aiohttp
>>> from python_awair import AwairLocal
>>> async def data():
...     async with aiohttp.ClientSession() as session:
...             client = AwairLocal(
...                     session=session, device_addrs=["AWAIR-ELEM-14175A.local"]
...             )
...             devices = await client.devices()
...             data = await devices[0].air_data_latest()
...             print(f"Device: {devices[0]}")
...             for sensor, value in data.sensors.items():
...                     print(f"  {sensor}: {round(value, 2)}")
...             print(f"  temperature again: {round(data.sensors.temperature, 2)}")
...
>>> asyncio.run(data())
Device: <AwairDevice: uuid=awair-element_4922 model=Awair Element>
  dew_point: 11.01
  abs_humid: 9.67
  co2_est: 504
  voc_baseline: 2461373767
  voc_h2_raw: 26
  voc_ethanol_raw: 37
  pm10_est: 1
  temperature: 20.61
  humidity: 54.11
  carbon_dioxide: 713
  volatile_organic_compounds: 159
  particulate_matter_2_5: 0
  temperature again: 20.61
>>>
```
🎉 🎉 🎉
